c-pallets/segment-book/src/lib.rs

```diff
@@ -436,7 +443,8 @@ pub mod pallet {
 		) -> DispatchResult {
 			let sender = ensure_signed(origin)?;
 			let acc = Self::get_current_scheduler()?;
-			if prove_info.len() > 100 {
+			if prove_info.len() > 100 {  // Why 100? Move hardcoded value to pallet const, Config type or Storage
+				// Also the same constant shuld be used for your benchamrks
```

Comment:
After our rough test, the time for calculating 100 proofs can be kept within 1s, and since we will uniformly submit the proofs submitted in the current block to the issuer of the block for verification, in order to avoid one issuer undertaking too many verification tasks, we have not set too large a value.

Reply:
ok, but in any case better to have is a const or part of the pallet configuration.

Reply:
and use the same one for bechmars

Reply:
OK, I have configured a constant for it:
https://github.com/CESSProject/cess/pull/88/files#diff-e315e38811b02c01aead9143bcfe8a7d0db178f24a154bad1431a08648393296R449-R450